### PR TITLE
Modified type definitions for `FileError.code`

### PR DESCRIPTION
**Pull request by jomarquez21**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [x] refactoring / style
- [ ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**

Allow exposing error codes in a constant.

**Does this PR introduce a breaking change?**

No.

**Other information**

### CODE REVIEW

**Comment by jomarquez21** [comment]
Any update?

**Comment by jomarquez21** [comment]
Any update?

**Comment by hieusmiths** [comment]
> Any update?

Hmm. I don't see any active on the project.

**Review by rolandjitsu — APPROVED** [review]

File: typings/react-dropzone.d.ts [hunk 1]
```diff
@@ -8,9 +8,16 @@ export interface DropzoneProps extends DropzoneOptions {
   children?(state: DropzoneState): JSX.Element;
 }
 
+export enum ErrorCodes {
+ FILE_INVALID_TYPE = 'file-invalid-type',
+ FILE_TOO_LARGE = 'file-too-large',
+ FILE_TOO_SMALL = 'file-too-small',
+ TOO_MANY_FILES = 'too-many-files',
+}
+
```
Comment (rolandjitsu) [inline typings/react-dropzone.d.ts:17]:
```suggestion
export enum ErrorCode {
 FileInvalidType = 'file-invalid-type',
 FileTooLarge = 'file-too-large',
 FileTooSmall = 'file-too-small',
 TooManyFiles = 'too-many-files',
}

```

File: typings/react-dropzone.d.ts [hunk 2]
```diff
@@ -8,9 +8,16 @@ export interface DropzoneProps extends DropzoneOptions {
...
 export interface FileError {
   message: string;
-  code: "file-too-large" | "file-too-small" | "too-many-files" | "file-invalid-type" | string;
+  code: ErrorCodes | string;
```
Comment (rolandjitsu) [inline typings/react-dropzone.d.ts:20]:
```suggestion
  code: ErrorCode | string;
```

**Review by rolandjitsu — CHANGES_REQUESTED** [review]

**Comment by rolandjitsu** [comment]
@jochenberger I've requested some changes, please take a look. We can merge after that.

File: typings/react-dropzone.d.ts (hunk 1 above)
Comment (jomarquez21) [reply typings/react-dropzone.d.ts:17]: Constants are generally capitalized.

Why should we use pascalcase to set the `ErrorCode` options?

File: typings/react-dropzone.d.ts (hunk 2 above)
Comment (jomarquez21) [reply typings/react-dropzone.d.ts]: change applied.

File: typings/react-dropzone.d.ts (hunk 1 above)
Comment (rolandjitsu) [reply typings/react-dropzone.d.ts]: TypeScript documents it as such, see [enums](https://www.typescriptlang.org/docs/handbook/enums.html).

Comment (jomarquez21) [reply typings/react-dropzone.d.ts]: Good point, I did not have it in mind, thank you very much.

**Review by rolandjitsu — APPROVED** [review]

**Pull request merged**